dask/imperative.py

```diff
@@ -183,6 +183,25 @@ def get(dsk, keys, get=None, **kwargs):
     return get(dsk2, keys, **kwargs)
 
 
+def compute(*args, **kwargs):
+    """Evaluate several ``Value``s at once.
+
+    Examples
+    --------
+    >>> a = value(1)
+    >>> b = a + 2
+    >>> c = a + 3
+    >>> compute(b, c)  # Compute both simultaneously
+    (3, 4)
+    """
+    if len(args) == 1:
+        return args[0].compute(**kwargs)
```

Comment:
I do like having type-stable functions. However, I did this specifically for the following:

```
>>> a = value(1)
>>> b = a + 1
>>> c = a + 2
>>> rb, rc = compute(b, c)    # Works, is integer
>>> rb = compute(b)    # Works, is integer
```

However, I don't feel too strongly about this. You're correct that `.compute` can always be used for the single element case.